crates/relayer/src/worker.rs

```diff
@@ -132,13 +132,38 @@ pub fn spawn_worker_tasks<ChainA: ChainHandle, ChainB: ChainHandle>(
                     let link = Arc::new(Mutex::new(link));
                     let resubmit = Resubmit::from_clear_interval(packets_config.clear_interval);
 
-                    let packet_task = packet::spawn_packet_cmd_worker(
-                        cmd_rx,
-                        link.clone(),
-                        should_clear_on_start,
-                        packets_config.clear_interval,
-                        path.clone(),
-                    );
+                    let src_chain_config =
+                        config.chains.iter().find(|chain| chain.id == chains.a.id());
```

Comment:
Shall we rather output an error and return `(None, None)` if we can't find the chain config?